CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
 # Changelog
+## 1.55.0
+* Enhancement - Introduce optional jitter for task cleanup wait duration, configurable via ECS_ENGINE_TASK_CLEANUP_WAIT_DURATION_JITTER environment variable. In use case where there are large number of tasks being stopped at the same time, specifying this jitter can help avoid all the task cleanup happening at the same time (the latter could add pressure to the instance and as a result affect running tasks) [#2969](https://github.com/aws/amazon-ecs-agent/pull/2969)
```

Comment:
nit: it seems the convention is putting Feature above Enhancement, Enhancement above bug

Reply:
Oh, I will do a new one, thanks!